lib/stan.js

```diff
@@ -280,17 +290,28 @@ Stan.prototype.createConnection = function() {
       that.nc.publish(reply);
     });
 
+    that.pingInbox = nats.createInbox();
+    that.pingSubscription = that.nc.subscribe(that.pingInbox, function(msg) {
+      processPingResponse(that, msg);
```

Comment:
Instead of a function, the lib typically does a closure, and provides a returned function. See `Stan.prototype.processAck`